doc/changes.md

```diff
@@ -1,5 +1,98 @@
 # Changelog
 
+## Release v0.13.0 (2024/09/05)
+
+[![Digital Object Identifier for PyGMT v0.13.0](https://zenodo.org/badge/DOI/10.5281/zenodo.13679420.svg)](https://doi.org/10.5281/zenodo.13679420)
+
+### Highlights
+
+*
```

Comment:
Just a few ideas for potential highlights:
```suggestion
* Add section "Technical Reference"
* Add tutorial "Draping a dataset on top of a topographic surface"
* Add support and tutorial for using non-ASCII characters
```

Reply:
Thanks. Make some changes in 1b2e2ae793d01a91137264139c866feb9747d021.